From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Wed, 22 Nov 2023 08:52:47 +0100
Subject: [PATCH] openblas.pc: add prefix=

This is necessary for the PTXdist pc file mangling to work correctly.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 cmake/openblas.pc.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmake/openblas.pc.in b/cmake/openblas.pc.in
index 7e120af86609..4a192b84ddaf 100644
--- a/cmake/openblas.pc.in
+++ b/cmake/openblas.pc.in
@@ -1,3 +1,4 @@
+prefix=@CMAKE_INSTALL_PREFIX@
 libdir=@CMAKE_INSTALL_FULL_LIBDIR@
 libsuffix=@SUFFIX64_UNDERSCORE@
 includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
